rai/results.go

```diff
@@ -431,6 +448,58 @@ func (c listItemColumn[T]) Value(rnum int) any {
 	return c.Item(rnum)
 }
 
+// Represents several sub-columns of a `listColumn` that represent one column for a composite type (e.g. int128)
+type listSliceColumn[T any] struct {
```

Comment:
is this what we were calling `compositeColumn` on our call? I think I liked that name a bit better

Reply:
I didn't call it that because this isn't the only way values made by composing primitive types together are stored as columns. Not a good general preference, but here I think we should be clear about how the struct works vs what user-level functionality it's intended for, given the complexity of all the various "column" transformations going on